ostree-compliance-mode.spec

```diff
@@ -0,0 +1,58 @@
+%global dracutdir %(pkg-config --variable=dracutdir dracut)
+
+Name:           ostree-compliance-mode
+Version:        0.1
+Release:        1%{?dist}
+Summary:        ostree-compliance-mode is a tool designed to purge protected data, leaving the system in a modifyable state in compliance with GPLv3
+
+License:        GPLv2
+URL:            https://github.com/ericcurtin/%{name}
+Source0:        %{url}/archive/%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires: meson
+BuildRequires: gcc
+BuildRequires: ninja-build
+BuildRequires: ostree
+BuildRequires: rpm-ostree
+BuildRequires: python3-devel
+
+%description
+%{summary}
+
+%prep
+%autosetup
+%global debug_package %{nil}
+
+%build
+%meson
+%meson_build
+
+%check
+%meson_test
+
+%install
+%meson_install
+meson build --buildtype=release --prefix=$RPM_BUILD_ROOT%{_prefix};
+ninja -v -C build
+ninja -v -C build install
+chmod 4011 $RPM_BUILD_ROOT%{_bindir}/ostree-compliance-mode
+
+mkdir -p $RPM_BUILD_ROOT%{_prefix}/libexec
+cp ostree-compliance-mode-helper $RPM_BUILD_ROOT%{_prefix}/libexec/ostree-compliance-mode-helper
+chmod 100 $RPM_BUILD_ROOT%{_prefix}/libexec/ostree-compliance-mode-helper
+
+mkdir -p $RPM_BUILD_ROOT/etc
+currently_deployed_ref="fedora:fedora/35/aarch64/silverblue"
+echo "$currently_deployed_ref" > $RPM_BUILD_ROOT/etc/ostree-compliance-mode.conf
+chmod 600 $RPM_BUILD_ROOT%{_sysconfdir}/ostree-compliance-mode.conf
+
+%files
+%license LICENSE
+%doc README.md
+%{_bindir}/ostree-compliance-mode
+%{_sysconfdir}/ostree-compliance-mode.conf
+%{_prefix}/libexec/ostree-compliance-mode-helper
+
+%changelog
+* Thu Sep 8 2022 Ian Mullins<imullins@redhat.com> - 0.1-1
```

Comment:
Should be:

`* Thu Sep 8 2022 Ian Mullins <imullins@redhat.com> - 0.1-1`

a space is missing between name and <email>